master/static/srv/proto_get_trial_workloads.sql

```diff
@@ -48,7 +50,11 @@ workloads AS (
       t.end_time,
       v.end_time,
       c.end_time
-    ) AS end_time
+    ) AS end_time,
+    coalesce(
+      t.metrics,
+      v.metrics
```

Comment:
looks like `checkpoints_view`(also in this query) has both training metrics and validation metrics together. it feels a little weird that we are separating them out and then putting them back together client side.

Reply:
ok I remember what happened here -- I don't know the specifics of `checkpoints_view` so I might be off-base on that.  

But training, validation, and checkpoints are separate events being pulled in from their respective tables (trainings_vt, checkpoints_vt, validations_vt) into one workloads view here with no overlaps, i.e. `FULL JOIN ... ON false`. COALESCE (as used here already for total_batches and end_time) is essentially UNION-ing. Maybe the coalesce is needed for the `v.validation::jsonb AS validation, t.training::jsonb AS training, c.checkpoint::jsonb AS checkpoint` line? I've followed suit with metrics here for our table and for ORDER BY.